src/event.js

```diff
@@ -130,7 +130,7 @@ module.exports.initializeEvent = function initializeEvent(resourceType, name, op
         errorCode.ErrorCode.OK,
     ]);
     slsEvent.setResource(resource);
-    return { slsEvent, startTime };
+    return { slsEvent, startTime, resource };
```

Comment:
This is redundent (you can get the resource from the event)

Reply:
👍